content/docs/development/oidc-certification.md

```diff
@@ -0,0 +1,154 @@
+
+- A public issuer URL
+- At least two separate clients (with redirect URIs pointing to `https://www.certification.openid.net/test/a/YOUR_ALIAS/callback`).
+
+`YOUR_ALIAS` is an arbitrary string that MUST be unique to avoid interference with other test runs.
+
+The easiest way to run a public Dex instance is running one locally and exposing it using a [tunnel](https://github.com/anderspitman/awesome-tunneling).
+[Ngrok](https://ngrok.com/) is probably the most popular one. Personally, I prefer using [tunnelto.dev](https://tunnelto.dev/) these days.
+Alternatively, you can install Dex on an EC2 instance or a DigitalOcean droplet or anything that is publicly available.
+
+Here is a minimal configuration example for running Dex:
+
+```yaml
+issuer: https://dex.tunnelto.dev/dex
+
+storage:
+  type: memory
+
+web:
+  http: 0.0.0.0:5556
+
+oauth2:
+  # Automate some clicking
+  # Note: this might actually make some tests pass that otherwise wouldn't.
+  skipApprovalScreen: true
+
+connectors:
+  # Note: this might actually make some tests pass that otherwise wouldn't.
+  - type: mockCallback
+    id: mock
+    name: Example
+
+# Basic OP test suite requires two clients.
+staticClients:
+  - id: first_client
+    secret: 89d6205220381728e85c4cf5
+    redirectURIs:
+      - https://www.certification.openid.net/test/a/dex/callback
+    name: First client
+
+  - id: second_client
+    secret: 51c612288018fd384b05d6ad
+    redirectURIs:
+      - https://www.certification.openid.net/test/a/dex/callback
+    name: Second client
+```
+
+Save it in a file (eg. `config.yaml`) then launch Dex:
+
+```shell
+dex serve config.yaml
+```
+
+Then launch the tunnel:
+
+```shell
+tunnelto --subdomain dex --port 5556
+```
+
+You can verify Dex running by checking the discovery endpoint:
+
+```shell
+curl https://dex.tunnelto.dev/dex/.well-known/openid-configuration
+```
+
+### Running tests
+
+1. Open https://www.certification.openid.net/ in your browser
+1. Login with your Google or Gitlab account
+1. Click _Create a new test plan_
+1. Select _OpenID Connect Core: Basic Certification Profile Authorization server test_ as the test plan
+1. _Server metadata location_ should be **discovery**
+1. _Client registration type_ should be **static_client**
+1. Choose an alias (that you used in the redirect URIs above)
+1. Enter the discovery URL
+1. Enter the first client details in the _Client_ and _Second client_ sections
+1. Enter the second client details in the _Client for client_secret_post_ section
+1. Hit _Create test plan_
+1. Run through each test case, following all instructions given by individual cases.
+    * In order to pass certain cases, screenshots of OP responses might be required.
+
+### Last results
+
+Dex does not fully pass the Basic profile test suite yet. The following table contains the current state of test results.
+
+| Test Name                                                                    | Status      | Result  |
```

Comment:
Do we need to add comments for failed results? They were for the previous version of the table. I get that this table is just plain results, but statuses like `Interrupted, Review` do not show whether Dex passed the test.

Reply:
See my comment above. I want to take another stab at the test suite soon and run a more thorough set of checks.